src/components/KTabs/KTabs.vue

```diff
@@ -11,21 +11,21 @@
         class="tab-item"
         :class="{ active: activeTab === tab.hash }"
       >
-        <div
+        <component
```

Comment:
Not sure if this is something you'll want to address in this PR, but if all of the "tabs" are using anchors, it means its essentially a `role="navigation"` (or just a `nav`). It just happens to look like what is generally termed a "tab look and feel". (In regards to line 5 here which I can't make a comment specifically on that line because GH)

Reply:
If you are using the `role="navigation"` you'll probably want to let the user decide what the `aria-label` is.

Question: Would you consider a very simple `KNavigation` component that was just a `<nav><ul><li>` with some similar styling for the tabs (i.e. very minimal JS)?

I would guess a theoretical `KNavigation` would be an very simple component to write as you need about 5% of the JS you have in here. You would also then avoid complicating `KTabs`. I would guess that would also stop @adamdehaven sweating about possible breaking changes 😅 so a positive all round!

Reply:
I think this is getting out of scope for the PR. 

TBH the existing aria-label should be removed as it's generic and doesn't actually describe the content. "Tabs" doesn't label the type of content being displayed.

Reply:
I agree that KNavigation is out of scope for this PR.
As for `aria-label`, it is an attribute required for elements with `role="tablist"` and `role="navigation"`. Removing it will bite us back in the upcoming a11y audit for sure. Instead I suggest that we bind `aria-label` (when provided) to this element instead of topmost wrapper `div` and add a paragraph in docs mentioning that it would be a good idea to provide a custom value, or otherwise the default ("Tabs") will be used?

Reply:
Again, I still think out of scope for this PR. 

Can create a ticket to discuss for a future update

Reply:
Ok but if we are using `role=navigation` here make sure you remember to also remove all the other aria attributes and that aren't needed on `role=navigation`. Would also be good to remove or at least make sure that the custom events added to the `li`s here don't break any existing native functionality provided by using anchors.

Whilst discussing this (and this might also be out of scope so let me know if so), I think you might want to remove some of these for when "hide-panels" is true also, as in that case its not really a role="tablist" either.

Let me know the link for that ticket @adamdehaven and I'll catch up with that tomorrow 👍